appinventor/components/src/com/google/appinventor/components/common/YaVersion.java

```diff
@@ -515,7 +515,10 @@ private YaVersion() {
   // For YOUNG_ANDROID_VERSION 208:
   // - FILE_COMPONENT_VERSION was incremented to 3
   // - BLOCKS_LANGUAGE_VERSION was incremented to 33
-  public static final int YOUNG_ANDROID_VERSION = 208;
+  // For YOUNG_ANDROID_VERSION 209:
+  // - BALL_COMPONENT_VERSION was incremented to 7
+  // - IMAGESPRITE_COMPONENT_VERSION was incremented to 7
+  public static final int YOUNG_ANDROID_VERSION = 209;
```

Comment:
You don't need to bump `TOUNG_ANDROID_VERSION`. We'll bump it when we actually merge your change.